pkg/kubectl/resource_printer.go

```diff
@@ -414,7 +415,7 @@ func (h *HumanReadablePrinter) EnsurePrintWithKind(kind string) {
 
 // Handler adds a print handler with a given set of columns to HumanReadablePrinter instance.
 // See validatePrintHandlerFunc for required method signature.
-func (h *HumanReadablePrinter) Handler(columns []string, printFunc interface{}) error {
+func (h *HumanReadablePrinter) Handler(columns []string, wideColumns []string, printFunc interface{}) error {
```

Comment:
name the arg to make clear the wide columns are added to the normal columns in wide mode (they don't need to repeat columns in the two lists)